templates/workloads/partials/cf/efs-access-point.yml

```diff
@@ -0,0 +1,20 @@
+{{- if .Storage}}
+{{- if .Storage.AccessPoint}}
```

Comment:
huh so this is a different setting than? https://aws.github.io/copilot-cli/docs/manifest/backend-service/#access-point-id

If I'm understanding right, this is only populated for a Copilot created FS?

Reply:
That's right